election-day/cbeer/election_day.go

```diff
@@ -0,0 +1,42 @@
+package electionday
+
+import "fmt"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of inital votes.
+func NewVoteCounter(initialVotes int) *int {
+	iv := initialVotes
```

Comment:
Why do you need the extra var iv? You can simply return the address of initialVotes.